bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -14,6 +14,11 @@ parser.add_argument('--verbose', action='store_true')
 parser.add_argument('--statmap-files', nargs='+',
                     help="List of coinc files to be redistributed")
 parser.add_argument('--cluster-window', type=float)
+parser.add_argument('--censor-ifar-threshold', type=float,
+    help="If provided, set the foreground censor to be times around only"
+         "candidates more significant than the given IFAR threshold")
+parser.add_argument('--censor-window', type=float, default=0.1,
```

Comment:
in coinc_statmap this is called 'veto-window' so prefer to have the same name here.

Reply:
(in that code the help is `help='Time around each zerolag trigger to window out, [default=.1s]')` which I think is clearer)